python2.cc

```diff
@@ -96,6 +96,8 @@ void PythonPrinter<2>::findInterpHeadFallback() {
         auto image = o.second;
         auto &syms = image->commonSections->debugSymbols;
         for (auto sym : syms) {
+            if (sym.name.substr(0, 10) == "_PyRuntime")
```

Comment:
Version detection has been moved into separate methods which will give their respective warnings if pstack wasn't compiled for that python version. The methods return true on success, false otherwise